module/Activity/src/Controller/ActivityController.php

```diff
@@ -290,6 +291,7 @@ public function signupAction(): Response|ViewModel
 
         if ($request->isPost()) {
             $form = $this->signupService->getForm($signupList);
+            /** @var ParametersInterface $postData */
             $postData = $request->getPost();
```

Comment:
This structure is used in many places (not everywhere) so we might want to do this in more places in the future.

Reply:
Yes, but somehow these specific cases were problematic according to the static analysis